torch/csrc/distributed/rpc/py_rref.cpp

```diff
@@ -46,6 +47,55 @@ RRefForkData fromPyTuple(const py::tuple& pyTuple) {
 
   return RRefForkData(ownerId, rrefId, forkId, parent, typeStr);
 }
+
+TypePtr tryTypePyObjToTypePtr(const py::object& type_value) {
+  if (type_value.is(py::none())) {
+    return nullptr;
+  }
+  c10::QualifiedName type_qualified_name =
+      c10::QualifiedName(py::cast<std::string>(
+          py::module::import("torch.jit").attr("_qualified_name")(type_value)));
+  TypePtr type_ptr = jit::get_python_cu()->get_type(type_qualified_name);
+  TORCH_CHECK(
+      type_ptr != nullptr,
+      "Type, ",
+      type_qualified_name.qualifiedName(),
+      ", has not been registered in JIT CompilationUnit yet. "
```

Comment:
please don't throw this error message, `JIT CompilationUnit` is not something the user should know about. We should just say: 

`type_hint` could not be found, did you pass a valid interface type?

Reply:
Good suggestion. Updated.